sysutils/ddrescueview/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+
+name            ddrescueview
+version         0.4.5
+revision        1
+categories      sysutils aqua
+platforms       darwin
```

Comment:
`platforms darwin` is now implicitly defined by MacPorts. You can delete this line.